net/modemmanager/Makefile

```diff
@@ -0,0 +1,141 @@
+	+kmod-usb-net-qmi-wwan \
+	+kmod-usb-net-hso \
+	+kmod-usb-net-rndis \
+	+kmod-usb-net-sierrawireless \
+	+kmod-usb-acm \
+	+kmod-usb-wdm \
+	+kmod-usb-serial \
+	+kmod-usb-serial-option \
+	+kmod-usb-serial-qualcomm \
+	+kmod-usb-serial-sierrawireless \
+	+kmod-usb-serial-option \
+	+kmod-usb-serial-wwan
+endef
+
+define Package/modemmanager/description
+  ModemManager is a DBus-activated service which allows controlling mobile
+  broadband modems.
+endef
+
+CONFIGURE_ARGS += \
+	--prefix=/usr \
+	--sysconfdir=/etc \
+	--localstatedir=/var \
+	--with-qmi \
+	--with-mbim \
+	--without-polkit \
+	--without-udev \
+	--without-suspend-resume \
+	--with-systemdsystemunitdir=no \
+
+define Build/Prepare
+	$(call Build/Prepare/Default)
+	( cd "$(PKG_BUILD_DIR)"; \
+		echo -e "all:\ninstall:" >po/Makefile.in.in; \
+	)
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include/ModemManager
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/ModemManager/*.h $(1)/usr/include/ModemManager
+	$(INSTALL_DIR) $(1)/usr/include/libmm-glib
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/libmm-glib/*.h $(1)/usr/include/libmm-glib
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmm-glib.so* $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/ModemManager.pc $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/mm-glib.pc $(1)/usr/lib/pkgconfig
+endef
+
+define Package/modemmanager/install
+	$(INSTALL_DIR) $(1)/lib/udev/rules.d
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/lib/udev/rules.d/*.rules $(1)/lib/udev/rules.d
+
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/ModemManager $(1)/usr/sbin
+
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/mmcli $(1)/usr/bin
+
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmm-glib.so* $(1)/usr/lib
+
+	$(INSTALL_DIR) $(1)/usr/lib/ModemManager
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ModemManager/libmm-plugin-*.so* $(1)/usr/lib/ModemManager
+
+	$(INSTALL_DIR) $(1)/etc/dbus-1/system.d
+	$(CP) $(PKG_INSTALL_DIR)/etc/dbus-1/system.d/org.freedesktop.ModemManager1.conf $(1)/etc/dbus-1/system.d
+
+	$(INSTALL_DIR) $(1)/usr/share/dbus-1/system-services
+	$(CP) $(PKG_INSTALL_DIR)/usr/share/dbus-1/system-services/org.freedesktop.ModemManager1.service $(1)/usr/share/dbus-1/system-services
+
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/modemmanager.init $(1)/etc/init.d/modemmanager
+
+	$(INSTALL_DIR) $(1)/etc/hotplug.d/net
+	$(INSTALL_DATA) ./files/25-modemmanager-net $(1)/etc/hotplug.d/net
+
+	$(INSTALL_DIR) $(1)/etc/hotplug.d/tty
+	$(INSTALL_DATA) ./files/25-modemmanager-tty $(1)/etc/hotplug.d/tty
+
+	$(INSTALL_DIR) $(1)/etc/modemmanager
+	$(INSTALL_DATA) ./files/modemmanager.common    $(1)/etc/modemmanager/modemmanager.common
+
+	$(INSTALL_DIR) $(1)/lib/netifd/proto
+	$(INSTALL_BIN) ./files/modemmanager.proto $(1)/lib/netifd/proto/modemmanager.sh
+endef
+
+$(eval $(call RequireCommand,xsltproc, \
```

Comment:
This will definitely fail on the buildbots. Please create Host packages if needed or add PKG_BUILD_DEPENDS:=package/host